vulture_os/services/frontend/models.py

```diff
@@ -1533,15 +1533,17 @@ def test_conf(self, node_name):
             try:
                 global_config = Cluster().get_global_config()
                 cluster_api_key = global_config.cluster_api_key
-                infos = post("https://{}:8000/api/services/frontend/test_conf/".format(node_name),
+                infos = post(f"https://{node_name}:8000/api/services/frontend/test_conf/",
                              headers={'cluster-api-key': cluster_api_key},
                              data={'conf': test_conf, 'filename': test_filename, 'disabled': not self.enabled},
-                             verify=False, timeout=30).json()
+                             verify=X509Certificate.objects.get(pk=1).ca_filename(), timeout=30).json()
```

Comment:
Good idea, but dangerous!

Reply:
Dangerous in case we change the certificate une Nginx or change completely pki system yes I agree.
Or if write_conf failed to save the certificates on disk.
Do we accept such risks ?